pulsar/producer_partition.go

```diff
@@ -322,9 +331,8 @@ func (p *partitionProducer) internalSend(request *sendRequest) {
 				WithField("sequenceID", sequenceID).
 				WithField("properties", msg.Properties).
 				Error("unable to send single message")
```

Comment:
if fail to add batch builder,should n't relaese publishSemaphore and callback? and why Send individually not retry if add fail?